src/core/output-layout.cpp

```diff
@@ -581,6 +586,29 @@ struct output_layout_output_t
             wlr_output_set_custom_mode(handle, mode.width, mode.height, mode.refresh);
         }
 
+        if (!is_nested_compositor)
```

Comment:
Is this something wlroots doesn't support or why put this check?

Reply:
When I use `if  (1)` here, nested wayland backend does not start. it fails with:
```
[types/output/swapchain.c:109] Swapchain for output 'WL-1' failed test
```

Reply:
I think I fixed it without the special case.